ui/src/routes/admin/colors/index.tsx

```diff
@@ -123,6 +132,203 @@ const ColorsComponent = () => {
           ))}
         </CardContent>
       </Card>
+      <Card>
+        <CardHeader>
+          <CardTitle>Ecosystem colors</CardTitle>
+        </CardHeader>
+        <CardContent>
+          <Table>
+            <TableHeader>
+              <TableRow>
+                <TableHead>Language</TableHead>
+                <TableHead>Ecosystems</TableHead>
+              </TableRow>
+            </TableHeader>
+            <TableBody>
+              <TableRow>
+                <TableCell>JavaScript/Node.js</TableCell>
```

Comment:
I believe there's currently no order, but it probably makes sense to sort alphabetically by "Language" for a quicker lookup.